docs/README-cmake.md

```diff
@@ -19,17 +12,16 @@ The CMake build system is supported on the following platforms:
 * Android
 * Emscripten
 * RiscOS
+* Playstation 2
 * Playstation Vita
+* Nintendo 3DS
 
 ## Building SDL
 
-Assuming the source for SDL is located at `~/sdl`
+Assuming the source for SDL is located at `~/sdl`. 
 ```sh
-cd ~
-mkdir build
-cd build
-cmake ~/sdl
-cmake --build .
+cmake -S ~/sdl -B ~/build -DSDL_STATIC=ON -DSDL_SHARED=ON
+cmake --build build

```

Comment:
```suggestion
cmake --build ~/build

```

Reply:
Please don't suggest -DSDL_STATIC=ON, that was the whole point of disabling it by default.

Reply:
I assumed it was mostly done for lowering the default build time.
I removed the reference to "shared and static libraries" in the next line.

Reply:
No, it was done so people can upgrade SDL in the future for products that are no longer supported.